backend-api/infra/monitoring/canaryAlarms.yaml

```
@@ -0,0 +1,656 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Resources:
+  AsyncTokenErrorRateAlarm:
+    Condition: DeployCanaryAlarms
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmDescription: "The number of Async Token Lambda errors is greater than or equal to 10% of all function invocations."
+      AlarmName: !Sub "${AWS::StackName}-token-error-rate"
```

```suggestion
      AlarmName: !Sub "${AWS::StackName}-token-lambda-error-rate"
```
it would be good to get the resource type in all of these